content/en/docs/tasks/administer-cluster/dns-debugging-resolution.md

```diff
@@ -188,7 +188,7 @@ for more details on Cluster Federation and multi-site support.
 ## References
 
 - [DNS for Services and Pods](/docs/concepts/services-networking/dns-pod-service/)
-- [Docs for the DNS cluster addon](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/dns/README.md)
+- [Docs for the kube-dns DNS cluster addon](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/dns/kube-dns/README.md)
```

Comment:
The original is a 404 -- it was recently broken out into two folders, one for kube-dns and one for CoreDNS. Which is unfortunate also bc we should probably change this to point to CoreDNS for 1.11, since it will then be the default. Sigh. Noting as a TODO on my end.